book-examples/leptos/src/default/card.rs

```diff
@@ -0,0 +1,18 @@
+#[allow(clippy::module_inception)]
+mod card;
+
+use leptos::prelude::*;
+use leptos_router::{
+    components::{Outlet, ParentRoute, Route},
+    path, MatchNestedRoutes,
+};
+
+#[component(transparent)]
+pub fn CardRoutes() -> impl MatchNestedRoutes + Clone {
+    view! {
+        <ParentRoute path=path!("/card") view=Outlet>
+            <Route path=path!("/") view=card::CardDemo />
```

Comment:
Could you also add the `CardWithForm` example? You can comment out components that don't exist yet, the Yew book examples do the same.